lib/homework1.ex

```diff
@@ -1,18 +1,50 @@
 defmodule Homework1 do
   @moduledoc """
-  Documentation for `Homework1`.
+  This module implements the simple calculation function.
+  You can use the Homework1.calculate/1 function to add, remove, multiply and divide.
   """
 
   @doc """
-  Hello world.
+  Calculate the result of an expression:
 
   ## Examples
-
-      iex> Homework1.hello()
-      :world
+      iex> Homework1.calculate "1 + 1"
+      2.0
+      iex> Homework1.calculate "12 - 76"
+      -64.0
+      iex> Homework1.calculate "11 * 9"
+      99.0
+      iex> Homework1.calculate "11 / 2"
+      5.5
 
   """
-  def hello do
-    :world
+
+  @spec calculate(String.t()) :: number()
+  def calculate(expression) when is_binary(expression) do
+    expression
+    |> String.replace(" ", "")
+    |> Float.parse()
+    |> compute()
+  end
+
+  @spec compute({Float.t(), String.t()}) :: number()
```

Comment:
Don’t write stuff just for the sake of writing more stuff.

To my best knowledge, there is no [`t:Float.t/0`](https://hexdocs.pm/elixir/Float.html#types).  
```suggestion
  @spec compute({float(), String.t()}) :: number()
```